demo/demo.md

```diff
@@ -147,11 +147,11 @@ Use the `pattern` attribute to set custom input validation. This example also us
 Be sure to review this element's API docs for all related information.
 
 <div class="exampleWrapper exampleWrapper--flex">
-  <auro-input id="demo5" pattern="[a-z]{1,15}" required maxlength="15" spellcheck="false" customValidityBadInput="Only contain lowercase letters w/no spaces">
+  <auro-input id="demo5" pattern="[a-z]{1,15}" maxlength="15" spellcheck="false" setCustomValidityBadInput="Only contain lowercase letters w/no spaces">
```

Comment:
```suggestion
  <auro-input id="demo5" pattern="[a-z]{1,15}" spellcheck="false" setCustomValidityBadInput="Only contain lowercase letters w/no spaces">
```

We should remove the `maxlength` from all these examples as well since they are not strictly tied to the specific example.